server/src/test/java/io/spine/server/entity/storage/given/ColumnsTestEnv.java

```diff
@@ -53,6 +53,33 @@ public int getValue() {
         }
     }
 
+    /**
+     * This entity declares a {@linkplain #setSecretNumber(Integer) mutator method}, however
+     * doesn't declare a respective accessor method.
+     *
+     * <p>{@code ColumnReader} should not get confused and assume that the mutator method is
+     * a property, and, therefore, a potential column.
+     */
+    @SuppressWarnings("unused") // Reflective access
+    public static class EntityWithASetterButNoGetter extends AbstractEntity<String, Any> {
+
+        private Integer secretNumber;
+
+        protected EntityWithASetterButNoGetter(String id) {
+            super(id);
+        }
+
+        @Column
+        public Integer getFortyThree(){
```

Comment:
The presence of this `@Column` makes the test a bit confusing. Can we just drop it and assert that there are _no_ columns?